src/GafferOSL/OSLShader.cpp

```diff
@@ -555,14 +583,18 @@ Plug *loadStructParameter( const OSLQuery &query, const OSLQuery::Parameter *par
 		result = existingPlug;
 	}
 
-	loadShaderParameters( query, result, parameter->name.string() + "." );
+	/// \todo Should we support metadata on the children of a struct parameter?
```

Comment:
Hmm, first problem is that you can't put a period in the name of a metadata.  The name has to be a valid identifier.

I guess I could use an underscore?  Andrew was asking about trying to get this rolled out this morning.  I guess I should probably give up on that.